integration_tests/helpers/apply.ts

```diff
@@ -369,10 +373,10 @@ export default class ApplyHelper {
     cy.task('stubAcctAlerts', { person: this.person, acctAlerts: this.acctAlerts })
   }
 
-  private stubDocumentEndpoints() {
+  stubDocumentEndpoints(documents?) {
```

Comment:
Please type the `documents` parameter, the linter should cover this, not sure why its not working.